mm2src/coins/eth.rs

```diff
@@ -7013,6 +7058,14 @@ pub fn pubkey_from_extended(extended_pubkey: &Secp256k1ExtendedPublicKey) -> Pub
     pubkey_uncompressed
 }
 
+fn extract_gas_limit_from_conf(coin_conf: &Json) -> Result<EthGasLimit, String> {
+    if coin_conf["gas_limit"].is_null() {
+        Ok(Default::default())
```

Comment:
This is already handled by `serde(default)`

Reply:
if "gas_limit" is null serde returns `invalid type: null` error. I wanted to handle this differently (to return the default) than any bad values (to fail)